### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by tkimnguyen**

### CODE REVIEW

**Comment by tkimnguyen** [comment]
@svx

**Comment by svx** [comment]
@tkimnguyen You really think simple buildout is simpler than the Unified installer for beginner or do I read it wrong ?

The beauty of the UI is that is comes with a script which tells you which dependencies you are missing and even with a ncurses interface, which is IMHO 'easier' than the simple buildout.
Plus is comes with its own egg cache. :)

**Comment by tkimnguyen** [comment]
Yes but does the unified installer run on Windows?

File: README.rst [hunk 1]
```diff
@@ -1,20 +1,22 @@
 ======================
-Plone Example Buildout
+Plone Simple Buildout
 ======================
 
 A basic, cross-platform `Plone <https://plone.org>`_ buildout.
 
+This installation method is simpler than the `Plone unified installer <https://github.com/plone/Installers-UnifiedInstaller>`_ for UNIX-based platforms, which automates the process of installing required software and setting up a virtual environment.
+
```
Comment (svx) [inline README.rst]: This confuses at least me :)
If you are new and on UNIX, IMHO UI is better as in easier.

**Comment by svx** [comment]
@tkimnguyen UI was never written for M$, but there is not much of a difference only requirements.txt and ncurses, you can run the UI without ncurses so if you add the missing requirements yes it will run on windows

**Comment by tkimnguyen** [comment]
!?$&%#!?! then why are we not recommending the unified installer for Windows?

**Review by svx — APPROVED** [review]
Changed the beginning telling use that method on Windows use UI if you are a beginner on UNIX

**Pull request merged**